tensorflow/lite/core/subgraph.cc

```diff
@@ -169,7 +169,6 @@ Subgraph::~Subgraph() {
     TfLiteIntArrayFree(node.inputs);
     TfLiteIntArrayFree(node.outputs);
     TfLiteIntArrayFree(node.temporaries);
-    if (node.builtin_data) free(node.builtin_data);
```

Comment:
This introduces a memory leak...

Reply:
@jdduke , I replaced malloc with smart pointer (make_unique) so the free is not required as  when the variable goes out of scope this will automatically release the memory. Also it fixes the design issue where the interface allocates the memory and the platform releases it.

Let e know your opinion on this.